packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -115,6 +120,32 @@ function makeAugmentedSchema(
         },
     });
 
+    composer.createScalarTC({
```

Comment:
I had thought there might be some benefit to abstracting it in case there was a transform to be done, but after diving even deeper into the [spec](https://relay.dev/assets/files/connections-932f4f2cdffd79724ac76373deb30dc8.htm#sec-Edge-Types.Introspection), it says that using String is fine.

I'll change it in the next commit.